### PR TITLE
fix kepler-operator ci

### PR DESCRIPTION
**Pull request by vprashar2929**
This PR:
* Add build args while building the operator image.
* Update the operator version to match the regex in pullspec.
* Add step registry for testing Instance deployment.
* Remove unit test job.

### CODE REVIEW

**Comment by vprashar2929** [comment]
/pj-rehearse  pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-kepler-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-kepler-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-kepler-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-kepler-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-kepler-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-kepler-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-kepler-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-unit

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-kepler-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-kepler-operator-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-operator-v1alpha1-kepler-operator-e2e

File: ci-operator/step-registry/openshift-power-monitoring/install/kepler/OWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+approvers:
+- vprashar2929
```
Comment (danilo-gemoli) [inline ci-operator/step-registry/openshift-power-monitoring/install/kepler/OWNERS:2]: Are you the only one?

Comment (vprashar2929) [reply ci-operator/step-registry/openshift-power-monitoring/install/kepler/OWNERS:2]: Currently yes. But might change in future

**Review by sthaha — COMMENTED** [review]
/lgtm

**Pull request merged**